src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -22,3 +22,21 @@
                          [selection]="expandedRow">
   </cd-rgw-bucket-details>
 </cd-table>
+
+<ng-template #bucketSizeTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="(row.bucket_quota.max_size > 0); else noSizeQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.size_usage">
+  </cd-usage-bar>
+  <ng-template #noSizeQuota>{{row.size_usage | dimlessBinary}}</ng-template>
+</ng-template>
+
+<ng-template #bucketObjectTpl
+             let-row="row">
+<ng-template #noObjectQuota>{{row.object_usage | dimlessBinary}}</ng-template>
+<cd-usage-bar *ngIf="(row.bucket_quota.max_objects > 0); else noObjectQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.object_usage">
+  </cd-usage-bar>
+</ng-template>
```

Comment:
No 4 new columns then (2 text ones + 2 progress bars)?

Reply:
Yes, there are 4 new columns added(attached screenshot in commit description). Only for progress bar we need ng-template else the data is displayed

Reply:
Thanks for the clarification, @avanthakkar !